common/config.go

```diff
@@ -78,9 +67,6 @@ func LoadFromCommandLine(config *Config) (*Config, error) {
 
 	flagSet.StringVar(&config.ConfigFile, "conf-file", "", "File path of configuration file")
 
-	flagSet.StringVar(&config.PluginDir, "plugin-dir", config.PluginDir, "Docker plugin directory where driver socket is created")
-	flagSet.StringVar(&config.DriverName, "driver-name", config.DriverName, "Name of Infoblox IPAM driver")
```

Comment:
I don't think so we can create multiple instances of the plugin. I tried installing a plugin twice and it gives error.
```
$ docker plugin ls
ID                  NAME                        DESCRIPTION                       ENABLED
d9d00a5239f2        ishant8/ipam-plugin:1.1.0   Infoblox IPAM plugin for Docker   true

$ docker plugin install ishant8/ipam-plugin:1.1.0
Plugin "ishant8/ipam-plugin:1.1.0" is requesting the following privileges:
 - network: [host]
 - mount: [/etc/infoblox]
 - mount: [/var/run]
 - allow-all-devices: [true]
 - capabilities: [CAP_SYS_ADMIN CAP_NET_ADMIN]
Do you grant the above permissions? [y/N] y
Error response from daemon: plugin "ishant8/ipam-plugin:1.1.0" already exists
```

There is a option **--alias** while installing the plugin and it creates the plugin with different name. 
```
$ docker plugin install ishant8/ipam-plugin:1.1.0 --alias infoblox
Plugin "ishant8/ipam-plugin:1.1.0" is requesting the following privileges:
 - network: [host]
 - mount: [/etc/infoblox]
 - mount: [/var/run]
 - allow-all-devices: [true]
 - capabilities: [CAP_SYS_ADMIN CAP_NET_ADMIN]
Do you grant the above permissions? [y/N] y
1.1.0: Pulling from ishant8/ipam-plugin
32f625173489: Download complete
Digest: sha256:525ded62dd7bf6920b796d32f931808c9403a0a1ee76d07c8657d3bfacc3a1aa
Status: Downloaded newer image for ishant8/ipam-plugin:1.1.0
Installed plugin ishant8/ipam-plugin:1.1.0

$ docker plugin install ishant8/ipam-plugin:1.1.0 --alias infoblox-2
Plugin "ishant8/ipam-plugin:1.1.0" is requesting the following privileges:
 - network: [host]
 - mount: [/etc/infoblox]
 - mount: [/var/run]
 - allow-all-devices: [true]
 - capabilities: [CAP_SYS_ADMIN CAP_NET_ADMIN]
Do you grant the above permissions? [y/N] y
1.1.0: Pulling from ishant8/ipam-plugin
32f625173489: Download complete
Digest: sha256:525ded62dd7bf6920b796d32f931808c9403a0a1ee76d07c8657d3bfacc3a1aa
Status: Downloaded newer image for ishant8/ipam-plugin:1.1.0
Installed plugin ishant8/ipam-plugin:1.1.0

$ docker plugin ls
ID                  NAME                DESCRIPTION                       ENABLED
0170bd86a366        infoblox:latest     Infoblox IPAM plugin for Docker   true
5057272f83d7        infoblox-2:latest   Infoblox IPAM plugin for Docker   true
```

**But the use case of --alias is different**. It is used for the seamless upgrades and get rid of the version dependency. For eg if we install plugin using full name - `docker plugin install ishant8/ipam-plugin:1.1.0` and then create a network with this ipam-plugin - `docker network create --ipam-plugin ishant8/ipam-plugin:1.1.0 net-1` , then net-1 is bound to this version of plugin. Upgrading with the `docker plugin upgrade ..` will cause issues as this network needs to be deleted first. 

So I dont think we can support multiple instances of ipam plugin on same host.